src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Use just - https://developer.mozilla.org/ru/docs/Web/CSS/background
To combine them